ci_framework/roles/cifmw_ceph_spec/templates/initial_ceph.conf.j2

```diff
@@ -1,5 +1,10 @@
 [global]
 osd pool default size = 1
+{% if cifmw_ceph_spec_public_network %}
```

Comment:
+1, this matches what I had in mind, we can enroll nodes by using the `ssh_network` and both `public` and `cluster` network are passed through the `initial_ceph.conf` assimilated at bootstrap time.